interpreter/cling/lib/Interpreter/CIFactory.cpp

```diff
@@ -426,12 +435,19 @@ namespace {
         }
       }
 #else // _MSC_VER
+      const char* RootInclude = std::getenv("ROOT_INCLUDE");
```

Comment:
Handling of -isystem should exist; skipping the gcc invocation if -nostdinc++ is passed or the ROOT part of the compiler includes do not exist.